README.md

```diff
@@ -1,28 +1,44 @@
-## Eclipse Sirius
+## Eclipse Sirius Desktop
 
 Sirius enables the specification of a modeling workbench in terms of graphical, table or tree editors with validation rules and actions using declarative descriptions.
 
-For more details see [the project page](https://www.eclipse.dev/sirius) and [the main wiki page](https://wiki.eclipse.org/Sirius).
+For more details see [the project's website](https://eclipse.dev/sirius).
 
 ### Building
 
-The build uses [Tycho](http://www.eclipse.org/tycho/). To launch a complete build, issue
+The build uses [Tycho](https://github.com/eclipse-tycho/).
+To launch a complete build, issue:
 
-```
+```sh
 mvn clean package
 ```
 
 from the top-level directory. The resulting update-site (p2 repository) can be found in `packaging/org.eclipse.sirius.update/target/repository`.
 
 By default the build uses a Neon-based target platform. You can specify a different platform like this:
```

Comment:
```suggestion
By default the build uses a 2023-03 target platform. You can specify a different platform like this:
```

Reply:
Or, this sentence can be deleted, since you added the same information at the end of this chapter.